### ISSUE TITLE
v8.4 broken with webpack 4.6 on Windows 10

### ISSUE THREAD

**Issue by jlchereau** [postcss/autoprefixer#1029]
Previous version was working fine.

````
ERROR in ./styles/app.page.error.less (./node_modules/css-loader??ref--7-1!./node_modules/postcss-loader/lib!./node_modules/less-loader/dist/cjs.js??ref--7-3!./
styles/app.page.error.less)
Module build failed: TypeError: Cannot read property 'index' of undefined
    at Root.prev (X:\project\node_modules\postcss\lib\node.js:371:33)
    at Processor.disabled (X:\project\node_modules\autoprefixer\lib\processor.js:346:25)
    at Processor.disabled (X:\project\node_modules\autoprefixer\lib\processor.js:373:45)
    at Processor.disabled (X:\project\node_modules\autoprefixer\lib\processor.js:373:45)
    at X:\project\node_modules\autoprefixer\lib\processor.js:223:24
    at X:\project\node_modules\postcss\lib\container.js:188:28
    at X:\project\node_modules\postcss\lib\container.js:144:26
    at Rule.each (X:\project\node_modules\postcss\lib\container.js:110:22)
    at Rule.walk (X:\project\node_modules\postcss\lib\container.js:143:21)
    at X:\project\node_modules\postcss\lib\container.js:146:32
    at Root.each (X:\project\node_modules\postcss\lib\container.js:110:22)
    at Root.walk (X:\project\node_modules\postcss\lib\container.js:143:21)
    at Root.walkDecls (X:\project\node_modules\postcss\lib\container.js:186:25)
    at Processor.remove (X:\project\node_modules\autoprefixer\lib\processor.js:222:20)
    at plugin (X:\project\node_modules\autoprefixer\lib\autoprefixer.js:87:32)
    at LazyResult.run (X:\project\node_modules\postcss\lib\lazy-result.js:277:20)
    at LazyResult.asyncTick (X:\project\node_modules\postcss\lib\lazy-result.js:192:32)
    at processing.Promise.then._this2.processed (X:\project\node_modules\postcss\lib\lazy-result.js:231:20)
    at new Promise (<anonymous>)
    at LazyResult.async (X:\project\node_modules\postcss\lib\lazy-result.js:228:27)
    at LazyResult.then (X:\project\node_modules\postcss\lib\lazy-result.js:134:21)
    at Promise.resolve.then.then (X:\project\node_modules\postcss-loader\lib\index.js:149:8)
    at <anonymous>
 @ ./styles/app.page.error.less 2:14-188
 @ ./js/app.error.js
```

**Comment by ai** [comment]
Fixed in https://github.com/postcss/autoprefixer/pull/1028

**Comment by ai** [comment]
Released in 8.4.1
